src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9555,5 +9555,45 @@ class C
     };
 }".Replace("&nbsp;", "\u00A0"));
         }
+
+        [Fact, WorkItem(41022, "https://github.com/dotnet/roslyn/issues/41022")]
+        [Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task SpacingAfterAttribute()
+        {
+            var code = @"class C
+{
+    void M([My]string?[]?[] x)
+    {
+    }
+}";
+            var expectedCode = @"class C
+{
+    void M([My] string?[]?[] x)
```

Comment:
Should we add an option for this? I've seen a lot of COM and P/Invoke APIs that save horizontal space for parameter lists by leaving out spaces.